spanner/spanner_snippets/snippet.go

```diff
@@ -1894,7 +1918,351 @@ func createClientWithQueryOptions(ctx context.Context, w io.Writer, database str
 
 // [END spanner_create_client_with_query_options]
 
-func createClients(ctx context.Context, db string) (*database.DatabaseAdminClient, *spanner.Client) {
+// [START spanner_create_backup]
+
+func createBackup(ctx context.Context, w io.Writer, adminClient *spadmindb.DatabaseAdminClient, database, backupID string) error {
```

Comment:
Heads up -- our sample style guide says to create clients in the sample rather than passing them as arguments. I think this makes sense for now since it matches the style of the rest of the Spanner samples, but wanted to call it out here.

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#initialize-clients-and-services-in-every-sample

Reply:
Thanks for the heads-up. CC @skuruppu

Reply:
Thanks @tbpg, we'll be happy to do a separate PR to make the samples consistent with the style guide.